### DISCUSSION TITLE
Disable smooth scrolling

### DISCUSSION THREAD

**Discussion by Muhammad95959** [LazyVim/LazyVim#5073] [Q&A]
How can I disable the new smooth scrolling effect?

**Comment by dpetka2001** [comment]
It's mentioned in the [docs](https://github.com/folke/snacks.nvim?tab=readme-ov-file#-installation) how to enable/disable each snacks module.
```lua
{
  "folke/snacks.nvim",
  priority = 1000,
  lazy = false,
  opts = {
    -- your configuration comes here
    -- or leave it empty to use the default settings
    -- refer to the configuration section below
    scroll = { enabled = false },
  },
}
```

**Reply by folke** [reply]
Or as mentioned in the news, set `vim.snacks_animate = false` in your `options.lua`

**Reply by MitchRivet** [reply]
small correction: it's `vim.g.snacks_animate` (in case others stumble across)
